plugins/panels/model_evaluation/utils.py

```
@@ -0,0 +1,104 @@
+import json
+import zlib
+import numpy as np
+from fiftyone.operators.cache.serialization import (
+    auto_serialize,
+    auto_deserialize,
+)
+
+
+def compress_confusion_matrix(matrix: np.ndarray, use_zlib=True):
```

Ruff (0.14.7)

Boolean default positional argument in function definition

(FBT002)